docs/Gruntfile.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function (grunt) {

```

Comment:
Can we move these files into `guides` or even a `.` subdirectory? That's where the Markdown lives after all. Also, while it's a bit of nitpicking: two space indention and newline character at the end of file ;-P

Reply:
i'll move it